docs/keywords.md

```diff
@@ -32,7 +32,7 @@ Keyword | Description
   `discourse-segmenter` | if the implementation uses a discourse segmenter
```

Comment:
why is `tokenizer`, `chunker`, `discourse` not `-based`?

Reply:
I didn't use based because it would probably mean that the whole transformation is based on the tokenizer or chunker. But anyways, I have appended based now.